hugo/content/hw/_index.md

```diff
@@ -144,19 +144,17 @@ If you want to stop the timer, set bit 1 in `TK1_MMIO_TIMER_CTRL`.
 
 A standard UART (Universal Asynchronous Receiver/Transmitter)
 interface is used for sending and receiving bytes to a TKey device app
-via the interface microcontroller on the TKey. The UART default
-configuration is:
-
-| *Config*  | *Default* | *Comment*        |
-|-----------|-----------|------------------|
-| Data rate | 62500 bps | Divisor = 288    |
-| Data bits | 8         |                  |
-| Stop bits | 1         | 0, 1, 2          |
-| Parity    | none      | not configurable |
-
-
-All configuration, except for parity, can be configured by the device
-app. Note that the client app must set the same configuration.
+via the interface microcontroller on the TKey. The UART configuration is:
+- baudrate: 62500 bps
+- data bits: 8
+- stop bit: 1
+- parity: none
+
+Note that the client app must set the same configuration. Configuring
+the UART core's baudrate, bit rate and stop bits, in runtime, is
+deprecated and should not be altered by an app. This is to ensure
+compatibility and prevent mismatch between different TKeys, the
+correct baudrate is set during build.
```

Comment:
I think "during build" can be misunderstood. Perhaps "correct baudrate is set when building the bitstream" or something like that?

I also think it should be its own sentence.

Reply:
Fixed.